docs/index.rst

```diff
@@ -215,3 +216,45 @@ In dask-distributed, a ``Worker`` is a Python object and node in a dask
 computations. ``Jobs`` are resources submitted to, and managed by, the job
 queueing system (e.g. PBS, SGE, etc.). In dask-jobqueue, a single ``Job`` may
 include one or more ``Workers``.
+
+Viewing the Dask Dashboard
+--------------------------
+
+It is recommended to run a Jupyter notebook server on the HPC and specify the IP address
+and port number. Then connect to the server using a SSH tunnel from your local
+machine e.g. your laptop or desktop.
+
+Firstly, configure the Jupyter notebook on the HPC as given in the
+`Pangeo documentation <http://pangeo-data.org/setup_guides/hpc.html#configure-jupyter>`_.
+This is repeated here for completeness:
+
+.. code-block:: bash
+
+
+    $ jupyter notebook --generate-config
+    $ ipython
+    In [1]: from notebook.auth import passwd; passwd()
+    Enter password:
+
+Copy the password hash to the file ``~/.jupyter/jupyter_notebook_config.py`` at the
+line ``c.NotebookApp.password = u''``. Make the configuration file read only by you:
+``chmod 400 ~/.jupyter/jupyter_notebook_config.py``.
+
+Start a Jupyter notebook server on the HPC (it is best practice to run/submit this as a
+job to an interactive queue).
+
+.. code-block:: bash
+
+
+    $ jupyter notebook --no-browser --ip=x.x.x.x --port=8888
+
+Use SSH tunneling on your local machine to access the notebook
+
+.. code-block:: bash
+
+
+    $ ssh -N -L 8888:x.x.x.x:8888 username@hpc_domain
+
+Open the Jupyter notebook in a browser on your local machine at http://localhost:8888.
+The Dask Dashboard should now be accessible by clicking the link displayed after
+inspecting the ``client`` object.
```

Comment:
That I believe it won't work for most users. For example, I have to do either SSH tunneling on bokeh scheduler adress and port, or use nbserverproxy. Only SSH tunneling to the notebook won't make the Dashboard accessible to me. So there is at least another SSH tunneling to describe.

Also, we should probably add a more detailed description of what has been done in the notebook.